Assets/HoloToolkit-Examples/Sharing/SharingService/Scripts/UserNotifications.cs

```diff
@@ -35,17 +35,17 @@ private void Connected(object sender = null, EventArgs e = null)
             usersTracker.UserLeft += NotifyUserLeft;
         }
 
-        private static void NotifyUserJoined(User user)
+        private void NotifyUserJoined(User user)
         {
-            if (user.IsValid() && localUser != user)
+            if (user.IsValid() && localUser.GetID() != user.GetID())
             {
                 Debug.LogFormat("[User Notifications] User {0} has joined the room.", user.GetName());
             }
         }
 
-        private static void NotifyUserLeft(User user)
```

Comment:
Why remove static?

Reply:
Hmm, why should static be there? The actual UserJoined and UserLeft events on SessionUsersTracker is not a static class. This makes it more inlined with the well-behaved RemoteHeadManager example class.